arrows/klv/klv_0104.h

```diff
@@ -7,11 +7,12 @@
  * \brief This file contains the interface for klv 0104 video metadata.
  */
 
-#ifndef KWIVER_VITAL_KLV_0104_H_
-#define KWIVER_VITAL_KLV_0104_H_
+#ifndef KWIVER_ARROWS_KLV_KLV_0104_H_
+#define KWIVER_ARROWS_KLV_KLV_0104_H_
+
+#include <arrows/klv/kwiver_algo_klv_export.h>
+#include <arrows/klv/klv_key.h>
```

Comment:
```suggestion
#include <arrows/klv/klv_key.h>
#include <arrows/klv/kwiver_algo_klv_export.h>
```

Reply:
Clarification: is this only for the purposes of alphabetization, or is there a functional reason?

Reply:
Lexicographical sorting, yes. (Note [[include.order]](https://github.com/Kitware/kwiver/blob/master/doc/code-style.rst#includes-include).)